### ISSUE TITLE
Customization of markdown renderer and editor

### ISSUE THREAD

**Issue by Uzay-G** [archivy/archivy#209]
Linked to https://github.com/archivy/archivy/issues/203#issuecomment-775593281.

Would be nice for users to be able to customize the markdown plugins / renderer used and with which options.

Ideas on how we could structure this would be really useful because I'm not sure what the best design would be.

**Comment by edditler** [comment]
It would probably be best to include more sections in `config.yml` next to the search configuration and allow users to change those settings from within the app. The simplest settings are these:

```
var md = require('markdown-it')({
  html:         true/false,        // Enable HTML tags in source
  xhtmlOut:     true/false,        // Use '/' to close single tags (<br />).
  breaks:       true/false,        // Convert '\n' in paragraphs into <br>
  linkify:      true/false,        // Autoconvert URL-like text to links
  typographer:  true/false,
  highlight:    function (/*str, lang*/) { return ''; }
```

What makes it a bit messy is that this config has to be passed to each template or put into a global variable like `session.config`.

**Comment by BrianGilbert** [comment]
#195 related

**Comment by Uzay-G** [comment]
This is fixed in #241 !
